components/notebook-controller/config/rbac/role.yaml

```diff
@@ -29,7 +29,6 @@ rules:
   - get
   - list
   - watch
-  - delete
```

Comment:
why this change to not allow deleting pods?

Reply:
sorry , this just WIP
it would be added back ,  updating it